src/repr/src/adt/mz_acl_item.rs

```diff
@@ -51,6 +51,8 @@ const CREATE_ROLE_CHAR: char = 'R';
 const CREATE_DB_CHAR: char = 'B';
 // compute Node
 const CREATE_CLUSTER_CHAR: char = 'N';
+// compute Network Policy
+const CREATE_NETWORK_POLICY_CHAR: char = 'P';
```

Comment:
how did you select `P`, feels like something we should run by the sql council? Also note the comment on the other `const`s capitalizes the letter in the resource that we're using here, e.g. this comment should be `// network Policy` if we use `P`

Reply:
yeah, well it is the P in network Policy, whether that's good enough justification IDK, but it seems as reasonable as anything I can think of.

Reply:
Yeah it seems quite reasonable, we could also do `n` but that might confuse users with `N`, sticking with `P` sounds good to me